.github/workflows/build.yml

```diff
@@ -40,6 +40,21 @@ jobs:
         with:
           path: ~/.platformio
           key: ${{ runner.os }}-esp8266-${{ hashFiles('platformio*.ini') }}
+      - uses: actions/cache@v2
+        if: ${{ contains(matrix.env, 'esp8285') }}
+        with:
+          path: ~/.platformio
+          key: ${{ runner.os }}-esp8285-${{ hashFiles('platformio*.ini') }}
```

Comment:
@TD-er I just found out that only ESP8266 and ESP32 environments where built. Added `ESP8285`, `WROOM02` and `hard` environments (there is 1 duplicate in these sets... not sure if that'll fail). We could opt to merge the ESP8266 and ESP8285 sets into 1, I think, by checking for esp82 instead of esp8266/esp8285.
If the `if:` condition (I assume it's a python-kind of expression?) supports OR functionality, also the wroom02 set could be added there.

Reply:
`'esp82'` sounds like a good compromise :)
Really glad you're looking into this so thoroughly 👍

Reply:
Hm, looks can be deceiving, at closer look the ESP8285, WROOM02 and 'hard' environments _are_ in there, I'll revert this change as it doesn't really add anything else than confusion.
That stale 'dev' environment is still there, haven't been able to find yet where that comes from 🤔